client/extensions/woocommerce/woocommerce-services/views/label-settings/label-payment-method.js

```diff
@@ -79,7 +83,15 @@ const PaymentMethod = ( {
 				checked={ selected }
 				onChange={ onSelect }
 			/>
-			<PaymentLogo className="label-settings__card-logo" type={ typeId } />
+			<PaymentLogo
+				className="label-settings__card-logo"
+				type={ typeId }
+				altText={ translate( 'Payment %(typeName)s', {
```

Comment:
Same with this, actually – since you have "VISA"/etc before the card number, this is redundant information.